packages/shared-base/src/utils/getLocalImplementation.tsx

```diff
@@ -22,15 +22,15 @@ export async function getLocalImplementation<T extends object>(
     RPCCache.set(ref, localImpl)
 
     const result: any = await impl()
-    for (const key in localImpl) {
+    for (const key of Object.keys(localImpl) as Array<keyof typeof localImpl>) {
         if (!Reflect.has(result, key)) {
             delete localImpl[key]
-            isUpdate && console.log(`[HMR] ${name}.${key} removed.`)
+            isUpdate && console.log(`[HMR] ${name}.${String(key)} removed.`)
```

Comment:
You cannot get symbol keys from Object.keys. No need to convert.

Reply:
<img width="424" alt="CleanShot 2022-05-17 at 12 26 17@2x" src="https://user-images.githubusercontent.com/3842474/168728807-b00480f4-6ec1-467c-ae29-bcabf30d52e4.png">

Reply:
because you cast the type to `keyof T`. It should be `keyof T & string`.